packages/deploy/README.md

```diff
@@ -0,0 +1,131 @@
+# Platform Deployment Client
+
+Platform Deployment Client allows you to deploy contracts through the OpenZeppelin Platform, manage deployment configuration and manage block explorer api keys.
+
+## Install
+
+```bash
+npm install platform-deploy-client
+```
+
+```bash
+yarn add platform-deploy-client
+```
+
+## Usage
+
+Start by creating a new _Team API Key_ in Defender, and granting it the capability to manage sentinels. Use the newly created API key to initialize an instance of the Sentinel client.
+
+```js
+const { PlatformClient } = require('platform-deploy-client');
+const client = PlatformClient({ apiKey: API_KEY, apiSecret: API_SECRET });
```

Comment:
In other packages we are using `new` keyword to instantiate client. We should be consistent.

Reply:
Since this package will become all encompassing and eventually housing many other clients I decided it might be best to create a helper function to instantiate multiple clients with one call, rather than instantiating each individual package with credentials. I will export the new clients so users still have the option to do so that way. I will also update the read me to make it clear you can use the helper function or instantiate individually.